engine/dlib/src/remotery/lib/Remotery.c

```diff
@@ -5265,13 +5227,6 @@ static rmtU32 ThreadProfiler_GetNameHash(ThreadProfiler* thread_profiler, rmtMes
     // Have to recalculate and speculatively insert the name every time when no cache storage exists
     name_len = strnlen_s(name, 256);
     name_hash = _rmt_HashString32(name, name_len, 0);
-
-// DEFOLD
-    if (name_hash == 0 || name_len == 0) {
-        dmLogError("REMOTERY: MAWE: ThreadProfiler_GetNameHash(b): The hash is 0x%08x! String is '%s' len: %u", name_hash, name, (uint32_t)name_len);
-    }
-// END DEFOLD
-
```

Comment:
I don't think we ever got any logs with this output in it, so I agree we should remove this now.

Reply:
Ah, yes, I wasn't sure if we should keep these logs or not?